src/client.ts

```diff
@@ -41,7 +41,7 @@ export type ScubaMetrics = {
     metricsClass: string;
     resourceName: string;
     id?: number;
-    date?: string;
```

Comment:
@williamlardier just making sure: Is this an issue for Artesca ?
( scuba was not returning a date, and after https://github.com/scality/scuba/pull/184 it will return one for all `getMetrics` and `getMetricsLatest` )